travis-ymls/high-scale-lib.travis.yml

```diff
@@ -0,0 +1,29 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : high-scale-lib
+# Source Repo         : https://github.com/stephenc/high-scale-lib.git
+# Travis Job Link     : https://travis-ci.com/github/rajesh-ibm-power/high-scale-lib
+# Created travis.yml  : Yes
+# Maintainer          : Rajesh kumar <Rajesh.kumar13@ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ---
+
+language: java
+jdk:
+  - openjdk11
+  
+dist: xenial
+
+arch:
+  - amd64
+  - ppc64le
+
+addons:
+  apt:
+    packages:
+     - maven
```

Comment:
No need to install maven. @rajesh-ibm-power

Reply:
@Siddhesh-Ghadi 
Updated with removal of maven addon